openassessment/xblock/static/js/src/lms/components/WaitingStepList.jsx

```diff
@@ -15,10 +34,53 @@ const WaitingStepList = ({ studentList, refreshData }) => {
     <Button onClick={() => refreshData()}>{gettext('Refresh')}</Button>
   );
 
+  const FindLearnerAction = (props) => {
+    const { tableInstance: { selectedFlatRows = [] } } = props || { tableInstance: {} };
+
+    const selectedFlatRowsLength = selectedFlatRows.length;
+
+    if (selectableLearners && selectedFlatRowsLength) {
+      const [row] = selectedFlatRows;
```

Comment:
Same comment here with the naming, I think we can do better than `row`

Reply:
Any idea? I can use rowStudent or rowTable